src/omega_moderne_client/campaign/campaigns/http_in_maven_pom/pr_message.md

```diff
@@ -12,7 +12,7 @@ This leaves your build vulnerable to allowing a [Man in the Middle](https://en.w
 This vulnerability has a CVSS v3.0 Base Score of [8.1/10](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:H).
 
 [POC code](https://max.computer/blog/how-to-take-over-the-computer-of-any-java-or-clojure-or-scala-developer/) has existed since 2014 to maliciously compromise a JAR file in-flight.
-MITM attacks against HTTP are [increasingly common](https://security.stackexchange.com/a/12050), for example [Comcast is known to have done it to their own users](https://thenextweb.com/insights/2017/12/11/comcast-continues-to-inject-its-own-code-into-websites-you-visit/#).
+MITM attacks against HTTP are [increasingly common](https://security.stackexchange.com/a/12050), for example [Comcast is known to have done it to their own users](https://thenextweb.com/news/comcast-continues-to-inject-its-own-code-into-websites-you-visit).
 
 This contribution is a part of a submission to the [GitHub Security Lab](https://securitylab.github.com/) Bug Bounty program.
```

Comment:
Is this needed?

Reply:
You'd be surprised. I've gotten arguments from some that it's not a common attack vector.

Reply:
Sorry, I just means the GitHub Bug Bounty program reference. :-)

Reply:
Oh, yes, that. I should can that now. It's not relevant anymore.

Reply:
Resolved! Much appreciated!